cpp/mail_krimdok_newsletter.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -o errexit -o nounset
+
+
+BACKUP_DIR=/usr/local/var/lib/tuelib/sent_newsletters
+mkdir --parents "$BACKUP_DIR"
+REPLY_TO=johannes.ruscheinski@uni-tuebingen.de
+SQL_QUERY="SELECT 'johannes.ruscheinski@uni-tuebingen.de'"
+declare -i count=0
+for regular_file in $(ls -p | grep -v /); do
+    bulk_mailer "$regular_file" \
+                "$SQL_QUERY" \
+                /usr/local/var/lib/tuelib/auxillary_newsletter_email_addresses \
```

Comment:
Where does this file come from? And wouldn't it be better to prefix it with krimdok_ or relbib_ etc.

Reply:
It comes from the FID directory on the network drive and there can nn conflict right now also because KrimDok systems have no overlap w/ anything else.